### DISCUSSION TITLE
Add setting for confirm construction (for touch)

### DISCUSSION THREAD

**Discussion by marvin1099** [OpenRCT2/OpenRCT2#23077] [Feature requests]
When using openrct on something like a phone and you want to place something like a flat ride, then it will place the ride were yo tap on the screen right away.
But since many phone screens are not super big it would be nice if it would not place the ride or whatever right away but only show the footprint like when you hover with the mouse.
And then to confirm placement have a tab be needed on the construction window.
This should probly a option that can be enabled.
Let me show a screenshots to show what I mean.
![Screenshot_20241028-225737_OpenRCT2](https://github.com/user-attachments/assets/5ce6d331-6ac8-4a0b-ba82-beaf06d051fc)
This I how it should look after you have tapped on the screen (this can be gotten on a touch screen if you start tapping outside the parkborder and then drag the building in the park) and then tapping on the marry go round icon would build it.

Edit:
This actually is almost a feature that is implemented, 
realy the start of the tap on a touch should be igoreable in when in Build mode
(since there is no hover on phone touch).

**Comment by Gymnasiast** [comment]
This is something we want, but it is much more involved than it may seem.

**Reply by marvin1099** [reply]
Well I hope a solution is found in the near future
